drv/gimlet-hf-server/src/main.rs

```diff
@@ -388,6 +395,34 @@ impl idl::InOrderHostFlashImpl for ServerImpl {
         poll_for_write_complete(&self.qspi);
         Ok(())
     }
+
+    fn get_mux(
+        &mut self,
+        _: &RecvMessage,
+    ) -> Result<HfMuxState, RequestError<HfError>> {
```

Comment:
Hmmm. This should probably not return `Result`. Idol support for that might be missing though -- did you try it?

Reply:
My disposition has always been to return a `Result` just to avoid changing an interface when a failure mode is discovered, but perhaps that's not warranted here.